snaps/features/custom-ui/with-jsx.md

```diff
@@ -467,6 +557,43 @@ module.exports.onHomePage = async () => {
 <img src={require("../../assets/custom-ui-heading.png").default} alt="Divider UI example" width="450px" style={{border: "1px solid #DCDCDC"}} />
 </p>
 
+### `Icon`
+
+Outputs an icon.
+
+#### Props
+
+- `name` - The name of the icon.
+  Possible values include `"confirmation"`, `"search"`, "`warning`", and `"menu"`.
+  See the [full list of possible `name` values](https://github.com/MetaMask/snaps/blob/0014ff6d7566dbc2945600db740d0a90f818b2d8/packages/snaps-sdk/src/jsx/components/Icon.ts#L5-L163).
```

Comment:
Listing out all the possible values looks a little crazy on the site:

![Screenshot 2024-09-25 at 12 00 15 PM](https://github.com/user-attachments/assets/c41a99cc-2773-4cd1-946c-0b5c7b35eaf3)

Instead, I replaced it with a link to the source file, and listing a few possible options. Also, does this prop have a default value?

Reply:
Thank you. There's no default value. The icon name has to be provided